tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -387,6 +397,28 @@ func setStoreWeightCommandFunc(cmd *cobra.Command, args []string) {
 	})
 }
 
+func setStoreHotWeightCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 3 {
+		cmd.Usage()
+		return
+	}
+	hotReadWeight, err := strconv.ParseFloat(args[1], 64)
+	if err != nil || hotReadWeight <= 0 {
```

Comment:
What does it mean if it equals zero? disable this store for hot region scheduler

Reply:
I think we should only allow weight > 0